editor/src/components/canvas/controls/outline-control.tsx

```diff
@@ -55,6 +56,48 @@ function isDraggingToMove(
   return dragState != null && dragState?.type === 'MOVE_DRAG_STATE' && targetIsDragged
 }
 
+interface CenteredCrossSVGProps {
+  id: string
+  style?: React.CSSProperties
```

Comment:
Unless we're intending to pass in custom styling here, this should probably just be replaced with `left` and `top` props

Reply:
done